docs/generating-a-provider.md

```diff
@@ -20,17 +20,11 @@ be quite similar for any other Terraform provider.
         export ProviderNameUpper=GitHub
         ```
 
-    2. Replace all occurrences of `template` with your provider name:
+    2. Run the `./hack/prepare.sh` script from repo root to prepare the repo, e.g., to 
```

Comment:
Looks like `hack/prepare.sh` wasn't committed.

Reply:
It lives in jet-template: https://github.com/crossplane-contrib/provider-jet-template/blob/main/hack/prepare.sh

Reply:
Oh sorry!